src/components/Select.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { SelectProps } from 'rendition';
+import styled, { withTheme } from 'styled-components';
+import asRendition from '../asRendition';
+import { radius } from '../theme';
+import { px, withProps } from '../utils';
+
+const Base = withProps<
+	{ emphasized?: boolean } & React.HTMLProps<HTMLSelectElement>
+>()(styled.select)`
```

Comment:
This should have ended up as `withProps<{ emphasized?: boolean }>()(styled.select)`. Did you face any issues?